src/io/flutter/actions/DeviceSelectorAction.java

```diff
@@ -67,9 +67,11 @@ public void update(final AnActionEvent e) {
 
     super.update(e);
 
-    Presentation presentation = e.getPresentation();
+    final Presentation presentation = e.getPresentation();
     if (!knownProjects.contains(project)) {
       knownProjects.add(project);
+      // TODO https://plugins.jetbrains.com/docs/intellij/disposers.html?from=jetbrains.org#choosing-a-disposable-parent
+      // Find a proper parent disposable. Projects and modules are never disposed.
```

Comment:
Would listening for project closing instead be better here?

Reply:
Good idea! Done.